### PR TITLE
Add test that validates the data pulled from cdo-donors

### PR DESCRIPTION
**Pull request by Hamms**
cdo-donors has a weight column whose maximum value must be 1.0 in order
for our weighted random select to work. Even though the gsheet that this
data is coming from has algorithms that should enforce this, it's gotten
messed up on occasion, and that mistake has in at least one case made it
through to production.

No longer!

### CODE REVIEW

File: pegasus/test/test_csv.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+class CSVTest < Minitest::Test
+  def test_cdo_donors_csv
+    donors_csv = CSV.read('./data/cdo-donors.csv')
+    max_donor_weight = donors_csv.last[4]
+    assert_equal max_donor_weight.to_f, 1.0
+  end
+
+  def test_cdo_donors_db
+    donors_with_max_weight = DB[:cdo_donors].where(weight_f: 1.0)
+    assert_equal donors_with_max_weight.count, 1
```
Comment (bcjordan) [inline pegasus/test/test_csv.rb:21]: I'm confused, there needs to be an entry with weight of `1.0`?

**Comment by bcjordan** [comment]
Code LGTM, could potentially add a file comment summarizing what's in this PR description

File: pegasus/test/test_csv.rb (hunk 1 above)
Comment (Hamms) [inline pegasus/test/test_csv.rb:21]: Precisely; we do a weighted select using that field on a randomly-generated number from 0 to 1.0. We've had a few errors where the top entry had a weight of < 1.0, so the randomly-generated number ended up selecting nothing for the weighted random selection.

**Comment by Hamms** [comment]
Sure thing! Thanks.

**Pull request merged**